code/game/gamemodes/miniantags/abduction/abduction.dm

```diff
@@ -459,16 +467,30 @@
 
 /datum/objective/abductee/engine
 	explanation_text = "Go have a good conversation with the Singularity/Tesla/Supermatter crystal. Bonus points if it responds."
-	
+
 /datum/objective/abductee/teamredisbetterthangreen
-	explanation_text = "Tell the AI (or a borg/pAI/drone if there is no AI) some corny technology jokes until it cries for help."
-	
-/datum/objective/abductee/time
-	explanation_text = "Go bug a bronze worshipper to give you a clock."
-		
+	explanation_text = "Tell the AI or other silicons some corny technology jokes until it cries for help."
+
 /datum/objective/abductee/licky
 	explanation_text = "You must lick anything that you find interesting."
-	
+
 /datum/objective/abductee/music
 	explanation_text = "Start playing music, you're the best musician ever. If anyone hates it, beat them on the head with your instrument!"
-	
+
+/datum/objective/abductee/clown
+	explanation_text = "The clown is not funny. You can do better! Make the crew laugh!"
```

Comment:
the thought of the clown being brainwashed into outdoing himself is hilarious imo